php81al2/versions.ini

```diff
@@ -1,14 +1,14 @@
 [php81]
-php = "8.1.8"
-curl = "7.83.0"
+php = "8.1.19"
+curl = "8.0.1"
 libjpeg = "v9e"
-libpng = "1.6.37"
+libpng = "1.6.39"
 libsodium = "1.0.18"
 libssh2 = "1.10.0"
-libxml2 = "2.9.12"
-libzip ="1.8.0"
-nghttp2 ="1.47.0"
-openssl = "1.1.1o"
-postgres = "14.4"
-redis = "5.3.7"
+libxml2 = "2.11.3"
+libzip ="1.9.2"
+nghttp2 ="1.53.0"
+openssl = "3.1.0"
```

Comment:
3.1.0 is not an LTS release, FYI. We should probably stick on 1.1.1t for now. Bref tried to upgrade to openssl 3.0.x, and encountered some performance regressions, so we should consider waiting that out, and staying on 1.1.x.